src/Init/Coe.lean

```diff
@@ -163,6 +163,7 @@ attribute [coe_decl] CoeOTC.coe
 
 instance [CoeOut α β] [CoeOTC β γ] : CoeOTC α γ where coe a := CoeOTC.coe (CoeOut.coe a : β)
 instance [CoeTC α β] : CoeOTC α β where coe a := CoeTC.coe a
+instance : CoeOTC α α where coe a := a
```

Comment:
We already had the reflexivity instance for `CoeTC`, this just adds them for the other auxiliary classes as well.